cmd/kube-scheduler/app/server.go

```diff
@@ -372,6 +373,7 @@ type SchedulerServer struct {
 	PodInformer                    coreinformers.PodInformer
 	AlgorithmSource                componentconfig.SchedulerAlgorithmSource
 	HardPodAffinitySymmetricWeight int32
+	CacheDebugResyncInterval       int32
```

Comment:
I think `CacheResyncInterval` may be a better name. Or perhaps `DebugCacheResyncInterval`. Having `Debug` in the middle of the name just feels strange to me.

In general, I think we should avoid adding debug-only options to components. It can easily lead to API bloat. What is the workflow for using this, what is it designed to debug, and what were the alternatives to extending your API with this parameter?

Reply:
I agree that we shouldn't add debug-only option, but alternatives are not great. One option is to add this to scheduler policy config. Another one is to hard-code the value and not make it configurable, which could be fine given that we have another mechanism to invoke the process.

Reply:
How about adding a simple bash script side-by-side to the scheduler container, which simply does `while true; do sleep 120; kill -12 $SCHED_PID; done`. Then we don't need to have this option exposed and keep the simplicity. 

@mtaufen some background, this debug behavior can be triggered by signal. This option simply adds periodical trigger.